hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSource.java

```diff
@@ -504,6 +504,15 @@ private void initialize() {
       }
     }
 
+    if (peerClusterId == null) {
```

Comment:
I think here we just need to add the same check like above?

```
    if (!this.isSourceActive()) {
      return;
    }
```

Reply:
Actually this makes more sense. Thanks for the idea.